### ISSUE TITLE
Mirror prebuilt Docker image to ECR Public

### ISSUE THREAD

**Issue by sorah** [aws/amazon-eks-pod-identity-webhook#141]
**What would you like to be added**: Have a image available on ECR Public

**Why is this needed**: To avoid rate limit against DockerHub

Related to https://github.com/aws/amazon-eks-pod-identity-webhook/issues/5, do you have a plan to publish Docker images currently only available on DockerHub to ECR Public? It would be convenient to avoid rate limit error from DockerHub.

**Comment by jwitko** [comment]
Any chance for traction on this issue?  This would be a huge improvement

**Comment by emattiza** [comment]
Id even entertain publishing under ghcr.io if thats acceptable to get around rate limits. ECR public would be the best, but happy to pr the build action here to support a ghcr.io target.

**Comment by dims** [comment]
@nckturner @emattiza looks like we can pick up this workflow from aws-ebs-csi-driver:
https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/.github/workflows/publish-ecr.yaml

and adapt it for us here?

**Comment by emattiza** [comment]
I agree @dims, and there are some makefile tweaks and a ./hack/provenance buildx test script. Happy to be the code mover if someone can handle the secret provisioning that will be support the PR.

**Comment by dims** [comment]
@emattiza please give it a shot! yea, i think we should be able to do the secret stuff
